.github/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 2. Bug - EGPH Delivery Controller frequency fixed - thanks to @kye-taylor (Kye Taylor)
 3. Enhancement - Update Agreements through TC NW for City,Southend departures - thanks to @cpawley (Chris Pawley)
 4. Bug - Fixed S10->East and Added S10->Montrose EGNT/NV inbound agreements - thanks to @hsugden (Harry Sugden)
+5. Bug - Corrected Galloway<->Talla agreements - thanks to @luketp (Luke Peters)
```

Comment:
```suggestion
5. Enhancement - Updated ScTMA Galloway<->Talla agreements - thanks to @luketp (Luke Peters)
```

Reply:
Being very picky, but think it makes sense to mark as an enhancement as these aren't published procedures and are just for event splits :)